AppKit/CPWindow/CPWindow.j

```diff
@@ -1958,6 +1964,74 @@ CPTexturedBackgroundWindowMask
 
         case CPLeftMouseDragged:
         case CPRightMouseDragged:
+            // First, we search for any tracking area requesting CPTrackingEnabledDuringMouseDrag.
+            // At the same time, we update the entered stack.
+            mouseEnteredStack = [];
+
+            for (var i = 0, nb = [_viewsWithTrackingAreas count], aView, trackingAreas, convertedPoint, event, trackingOptions; i < nb; i++)
```

Comment:
Please don't put anything other than loop-related vars in the initializer, it's hard to read. Make the assignments below a `var` block. Same for other places in the code.